CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## 0.12.21
 
+* Introduce test `retry` option and corresponding configuration. Also
+  introduce related `no-retry` runner flag.
```

Comment:
We should only support `--no-retry`. Passing `--retry` doesn't really mean anything.